.github/workflows/__shared-ci.yml

```diff
@@ -41,9 +41,51 @@ jobs:
       build: |
         {
           "commands": ["build"],
-          "artifact": ["packages/docs/build"]
+          "artifact": ["packages/docs/build", "packages/cli/dist"]
         }
 
+  package-test:
+    needs: [linter, nodejs]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+      - name: Setup Node.js
+        uses: hoverkraft-tech/ci-github-nodejs/actions/setup-node@1d00c9eb280acbee5df4b4a2087f786e66b13d87 # 0.14.1
+      - name: Download build artifacts
+        uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 # v4.1.8
+        with:
+          name: build
+          path: .
+      - name: Verify CLI build exists
+        run: |
+          echo "Checking if CLI binary exists..."
+          ls -la packages/cli/dist/bin/ || echo "CLI binary directory missing"
+          if [ ! -f "packages/cli/dist/bin/ci-dokumentor.js" ]; then
+            echo "CLI binary not found, building CLI package..."
+            npx nx build @ci-dokumentor/cli
+          fi
+          ls -la packages/cli/dist/bin/
+      - name: Create package tarball
+        run: |
+          cd packages/cli
+          npm pack --pack-destination ${{ runner.temp }}/
+      - name: Test package installation
+        run: |
+          mkdir -p ${{ runner.temp }}/test-install
+          cd ${{ runner.temp }}/test-install
+          npm init -y
+          npm install ../ci-dokumentor-cli-*.tgz
```

Comment:
I prefer to test with npx global installation and use ci-dokumentor-cli not full path